mobile/lib/pages/article_selection_page.dart

```diff
@@ -1,176 +1,37 @@
-import 'dart:async';
-
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
-import 'package:html_unescape/html_unescape.dart';
-import 'package:leancode_hooks/leancode_hooks.dart';
-import 'package:radioaktywne/components/utility/color_shadowed_card.dart';
-import 'package:radioaktywne/components/utility/image_with_overlay.dart';
-import 'package:radioaktywne/components/utility/ra_progress_indicator.dart';
-import 'package:radioaktywne/components/utility/ra_refresh_indicator.dart';
-import 'package:radioaktywne/extensions/extensions.dart';
+import 'package:radioaktywne/components/utility/lazy_loaded_grid_view.dart';
 import 'package:radioaktywne/models/article_info.dart';
-import 'package:radioaktywne/pages/ra_error_page.dart';
 import 'package:radioaktywne/resources/fetch_data.dart';
-import 'package:radioaktywne/resources/ra_page_constraints.dart';
 import 'package:radioaktywne/router/ra_routes.dart';
 
-class ArticleSelectionPage extends HookWidget {
+class ArticleSelectionPage extends StatelessWidget {
   const ArticleSelectionPage({
     super.key,
     this.timeout = const Duration(seconds: 15),
   });
 
   final Duration timeout;
 
+  static final Uri _articlesUri = Uri.parse(
+    'https://radioaktywne.pl/wp-json/wp/v2/posts?_embed=true',
```

Comment:
I suggest splitting the url into multiple consts, allowing future links to have the format of `BASE_URL+SUB_URL+ANOTHER_SUB_URL`